src/org/omegat/gui/editor/EditorPopups.java

```diff
@@ -76,6 +78,14 @@ public static void init(EditorController ec) {
 
     private EditorPopups() {
     }
+    
+    /** 
+     * Translate menu item but ensuring it does not contain mnemonics,
+     * because they do not work with context menus
+     **/
+    public static String translateMenu(String id) {
+        return OStrings.getString(id).replace("&","");
```

Comment:
This is not enough.
Please look
https://github.com/omegat-org/lib-mnemonics/blob/master/src/org/openide/awt/Mnemonics.java#L104-L108

Reply:
OK, have a look to the PR in lib-mnemonics, if you aprove it it would make possible to have all about mnemonics in same location (else I must copy the regex here)
Tell me what you think

Reply:
You can find a lib-mnemonics version 1.1 now.
https://central.sonatype.com/artifact/org.omegat/lib-mnemonics/1.1

Reply:
Corrected